packages/desktop/src/renderer/components/ContextMenu/ContextMenu.component.tsx

```diff
@@ -79,64 +76,66 @@ export const ContextMenu: FC<ContextMenuProps> = ({ visible, handleClose, title,
               alignItems: 'center',
               cursor: 'pointer',
             }}
-            onClick={handleClose}
+            onClick={handleBack || handleClose}
           >
             <Icon src={arrowLeft} />
           </Grid>
           <Grid style={{ flex: 5, justifyContent: 'center' }}>
             <Typography fontSize={16} fontWeight={'medium'} style={{ alignSelf: 'center' }}>
-              {title} settings
+              {title}
             </Typography>
           </Grid>
           <Grid style={{ flex: 1 }}></Grid>
         </Grid>
-        {hint && (
-          <Grid
-            style={{
-              width: '100%',
-              padding: 16,
-              borderTopWidth: 1,
-              borderColor: '#F0F0F0',
-            }}
-          >
-            <Typography fontSize={14} fontWeight={'normal'}>
-              {hint}
-            </Typography>
-          </Grid>
-        )}
-        <List>
-          {items.map((item, index) => {
-            return (
-              <Grid
-                style={{
-                  cursor: 'pointer',
-                  borderTop: '1px solid',
-                  borderColor: '#F0F0F0',
-                  borderBottomWidth: index === items.length - 1 ? '1px solid' : 0,
-                }}
-                key={index}
-              >
-                <ContextMenuItem {...item} />
-              </Grid>
-            )
-          })}
-        </List>
+        {children}
       </Grid>
     </Grid>
   )
 }
 
+export const ContextMenuHint: FC<ContextMenuHintProps> = ({ hint }) => {
+  return (
+    <Grid
+      style={{
+        width: '100%',
+        padding: 16,
+        borderTopWidth: 1,
+        borderColor: '#F0F0F0',
+      }}
+    >
+      <Typography fontSize={14} fontWeight={'normal'}>
+        {hint}
+      </Typography>
+    </Grid>
+  )
+}
+
+export const ContextMenuItemList: FC<ContextMenuItemListProps> = ({ items }) => {
+  return (
+    <List
+      style={{
+        padding: 0,
+        width: '100%',
+      }}
+    >
+      {items.map((item, index) => {
+        return <ContextMenuItem {...item} key={index} />
+      })}
+    </List>
+  )
+}
```

Comment:
I've separated these out to make the context menu more general purpose. Not all context menu views will have item lists.